src/main/java/com/batix/rundeck/core/AnsibleRunner.java

```diff
@@ -346,12 +346,9 @@ public int run() throws Exception {
     }
 
     if (debug == Boolean.TRUE) {
-      procArgs.add("-vvvv");

```

Comment:
changed from -vvvv to -vvv
level 4 verbosity is too much for a front end GUI, I've seen this crash Rundeck a few times. Level3 verbosity should be the extreme when viewing logs through the front end.